main.go

```diff
@@ -49,6 +50,18 @@ func main() {
 		*dbpath = ""
 	}
 
+	if *many == -1 {
+		if os.Getenv("HYDRA_MANY") != "" {
```

Comment:
For consistency, call this SYBIL_MANY?

Reply:
Hm.. or maybe HYDRA does make more sense since it is the name of the project

Reply:
Yeah, that makes more sense. I'm not super keen on "many". Shall we just rename this flag to `-nsybils` and have the env var as `NSYBILS`? This var is passed as [`NSybils` to Hydra](https://github.com/libp2p/hydra-booster/blob/master/main.go#L64) so then it's super consistent...

Reply:
Oh yeah I originally prefixed with `HYDRA_` to namespace it...`HYDRA_NSYBILS`?

Reply:
PERFECT! :D